client/client-multi/client-c/src/kaa/utilities/kaa_log.c

```diff
@@ -65,7 +65,12 @@ kaa_error_t kaa_log_create(kaa_logger_t **logger_p, size_t buffer_size, kaa_log_
     }
 
     (*logger_p)->buffer_size = buffer_size;
+
+#ifdef ESP8266_PLATFORM
+    (*logger_p)->sink = sink ? sink : 0;
+#else
     (*logger_p)->sink = sink ? sink : stdout;
```

Comment:
Maybe we can add the stdout definition to the platform-dependent to the stdio.h header?